sycl/test-e2e/Matrix/runtime_query_tensorcores.cpp

```diff
@@ -0,0 +1,122 @@
+// REQUIRES: cuda
+// RUN: %{build} -Xsycl-target-backend --cuda-gpu-arch=sm_70 -o %t.out
+// RUN: %{run} %t.out
+//
+// This tests the joint matrix runtime query for the cuda backend.
+// This test must be compiled with -Xsycl-target-backend --cuda-gpu-arch=sm_xx,
```

Comment:
I don't think this statement is actually true. I think that if you compile with default sm_50 the test will pass, even if you run it on e.g. sm_80.

Reply:
I removed the statement